### DISCUSSION TITLE
thanks and information

### DISCUSSION THREAD

**Discussion by 5a796d** [sonik-br/lgff_wheel_adapter#9] [General]
thanks for the project, I've been waiting for an adapter like this for a long time, finally my G923 (ps4) detects it as DFP, now I will be able to play the various F1 GPs again (2 and 3, 4 sees the G923) and Gran Turismo 4 with the steering wheel (I cry), and as soon as I can, I will also test it on Windows 98 and XP!

A request for information: I would be curious to know the precise latency times of the adapter, although I imagine they are acceptable

Thanks!!!!!

**Comment by sonik-br** [comment]
Hi!
Thanks for the feedback :)

I did not measured the latency...

The translating process from one wheel to another happens super fast on the pico.

All usb devices informs the interval that the host can poll it for more data.
The pico will poll the physical racing wheel at it's recommended interval. Each wheel model have it's specific interval.
The pico emulates wheels and uses informs a fixed value of 10ms interval.

Knowing this, the input latency will vary a lot depending on the wheel being used.

**Reply by 5a796d** [reply]
great thanks for the reply!

another piece of information, support for momo racing/momo force as output would be nice ;)

**Reply by sonik-br** [reply]
I can only add support for devices that I have access. I had to buy or borrow from friends most of the racing wheels.
I don't know anyone with a Momo. Do you happen to have one?

**Reply by 5a796d** [reply]
unfortunately not, thanks anyway, it's already a long time since I see it as a dfp!

**Comment by 5a796d** [comment]
I saw that you added initial support to momo force and momo racing, could you release uf2? Thank you!

**Reply by sonik-br** [reply]
Can you elaborate why momo output is useful?

**Reply by 5a796d** [reply]
> Can you elaborate why momo output is useful?

to be able to use the wheel with the original gp2, using the dfp version the wheel does not calibrate correctly in the game (also adjusting the steering degrees from the profiler) but with the momo racing I'm sure it works, I was playing it at the time

**Reply by sonik-br** [reply]
Ok. I'm not sure if it will work.
At least when testing with the modern drivers... The driver tries to read it in RAW mode. I can't implement that mode without access to a real momo wheel.
I've implemented it in HID mode and it works on a PS2 console with some games.
I can build a UF2 for you to try.

**Reply by 5a796d** [reply]
> Ok. I'm not sure if it will work. At least when testing with the modern drivers... The driver tries to read it in RAW mode. I can't implement that mode without access to a real momo wheel. I've implemented it in HID mode and it works on a PS2 console with some games. I can build a UF2 for you to try.

having released initial support I thought you had a chance to test it, thanks anyway

**Reply by sonik-br** [reply]
Have you tested with those output modes?
WingMan Formula GP, WingMan Formula Force GP, Driving Force.
They're older wheels and might be compatible with the game.
